driver/main.c

```diff
@@ -1820,6 +1812,10 @@ static int record_event_consumer(struct ppm_consumer_t *consumer,
 	ASSERT(ring);
 
 	ring_info = ring->info;
+	if(!ring_info) {
```

Comment:
We won't send any event on the newly enabled CPU; ring_info will be NULL and we would panic.